src/conversion_helper.js

```diff
@@ -314,6 +314,14 @@ const unitLookupList = [
       }
     },
     "ignoredUnits" : [/° ?C/, "degrees? c", "celsius", "kelvin"]
+  },
+  {
+    "imperialUnits" : [/nmi/, /nautical\smiles?/],
+    "standardInputUnit" : " nmi",
+    "isInvalidInput" : isZeroOrNegative,
+    "isWeaklyInvalidInput": isHyperbole,
+    "conversionFunction" : (i) => createMap(i * 1.852, "km"),
```

Comment:
There is a helper function in this file called `distanceMap()` that takes in a distance in meters and returns a map with an appropriate SI unit

Reply:
Cool beans, thanks for pointing that out.